src/Command/CreateSiteCommand.php

```diff
@@ -68,6 +69,7 @@ public function configure(): void
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $helper = $this->getHelper('question');
```

Comment:
This should work with phpstan symfony extension if the console is configured

Reply:
I will try to configure it.

Reply:
Fixed,